Easydict/Feature/ViewController/Window/BaseQueryWindow/EZBaseQueryViewController.m

```diff
@@ -367,7 +367,7 @@ - (EZQueryService *)defaultTTSService {
 }
 
 - (BOOL)isShowTipsView {
-    if (self.queryModel.actionType != EZActionTypeInputQuery && EZ_isEmptyString(self.queryModel.queryText) && !self.hasShowTips) {
```

Comment:
Why this removed `self.queryModel.actionType != EZActionTypeInputQuery`?

Reply:
Is it better to display the tips view when user use input translate? User may get hints from different places

Reply:
If user don't like the tips view they can close it in advance tab

Reply:
I fixed this https://github.com/tisfeng/Easydict/pull/418#issuecomment-1971438308 by adding a condition`self.queryModel.actionType != EZActionTypeInputQuery`

Reply:
okay, I know it. let me revert it